apis/checkpoint/finalized_state.yaml

```diff
@@ -0,0 +1,44 @@
+get:
+  operationId: "getFinalizedCheckpointState"
+  summary: "Get full BeaconState object for finalized checkpoint state"
+  description: |
+     Returns full BeaconState object for a finalized checkpoint state from the WS period.
```

Comment:
It is probably worth explicitly stating that the state should not have empty slots applied. That is it should be the state that is referenced by the block some finalized checkpoint root actually points to.